src/client/components/Purchases/Purchases.component.js

```diff
@@ -4,11 +4,13 @@ import './Purchases.styles.css';
 
 export default function Purchases({ orderId, date }) {
   return (
-    <form>
-      <label className="title">PURCHASES:</label>
-      <label className="orderid">ORDER ID: {orderId}</label>
-      <label className="date">DATE: {date}</label>
-    </form>
+    <div className="purchases">
+      <form>
+        <label className="title">PURCHASES:</label>
+        <label className="orderid">ORDER ID: {orderId}</label>
+        <label className="date">DATE: {date}</label>
```

Comment:
as In purchase all user's orders is showing here , so should not you  do a map on order object and show them as a list?